ui/app/templates/storage/index.hbs

```diff
@@ -0,0 +1,233 @@
+              </B.Td>
+              <B.Td data-test-csi-volume-controller-health>
+                {{#if B.data.controllerRequired}}
+                  {{if (gt B.data.controllersHealthy 0) "Healthy" "Unhealthy"}}
+                  (
+                  {{B.data.controllersHealthy}}
+                  /
+                  {{B.data.controllersExpected}}
+                  )
+                {{else if (gt B.data.controllersExpected 0)}}
+                  {{if (gt B.data.controllersHealthy 0) "Healthy" "Unhealthy"}}
+                  (
+                  {{B.data.controllersHealthy}}
+                  /
+                  {{B.data.controllersExpected}}
+                  )
+                {{else}}
+                  <em class="is-faded">
+                    Node Only
+                  </em>
+                {{/if}}
+              </B.Td>
+              <B.Td data-test-csi-volume-node-health>
+                {{if (gt B.data.nodesHealthy 0) "Healthy" "Unhealthy"}}
+                (
+                {{B.data.nodesHealthy}}
+                /
+                {{B.data.nodesExpected}}
+                )
+              </B.Td>
+              <B.Td data-test-csi-volume-provider>
+                {{B.data.provider}}
+              </B.Td>
+              <B.Td data-test-csi-volume-allocations>
+                {{B.data.allocationCount}}
+              </B.Td>
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+        {{#if (gt this.sortedCSIVolumes.length this.userSettings.pageSize)}}
+          <Hds::Pagination::Numbered
+            @totalItems={{this.filteredCSIVolumes.length}}
+            @currentPage={{this.csiPage}}
+            @pageSizes={{this.pageSizes}}
+            @currentPageSize={{this.userSettings.pageSize}}
+            @onPageChange={{action this.handlePageChange "csi"}}
+            @onPageSizeChange={{action (mut this.userSettings.pageSize)}}
+          />
+        {{/if}}
+      {{else}}
+        <div class="empty-message" data-test-empty-csi-volumes-list-headline>
+          {{#if this.csiFilter}}
+            <p>No CSI volumes match your search for "{{this.csiFilter}}"</p>
+            <Hds::Button @text="Clear search" @color="secondary" {{on "click" (queue (action (mut this.csiFilter) "") (action this.handlePageChange "csi" 1))}} />
+          {{else}}
+            <p>No CSI Volumes found</p>
+          {{/if}}
+        </div>
+      {{/if}}
+    </Hds::Card::Container>
+
+    <Hds::Card::Container @level="base" @hasBorder={{false}} class="storage-index-table-card">
+      <header aria-label="Dynamic Host Volumes">
+        <h3>Dynamic Host Volumes</h3>
+        <p class="intro">
+          Storage provisioned via plugin scripts on a particular client, modifiable without requiring client restart.
+          <Hds::Link::Inline @href="https://developer.hashicorp.com/nomad/docs/commands/volume/create" @icon="docs-link" @iconPosition="trailing">Read more</Hds::Link::Inline>
```

Comment:
It's not a published doc yet, but a better link might be to https://github.com/hashicorp/nomad/blob/main/website/content/docs/other-specifications/volume/host.mdx